src/Stakeholders/components/Agencies/NotificationForm/index.js

```diff
@@ -1,12 +1,11 @@
-import { httpActions } from '@codetanzania/emis-api-client';
+import { get } from '@codetanzania/emis-api-client';
```

Comment:
when is this used?

Reply:
it is used in this component SearchableSelectInput